backend/aka/clients/prisme.py

```diff
@@ -67,7 +71,7 @@ def __init__(self, claimant_id, cpr_cvr, external_claimant, claim_group_number,
         for file in files:
```

Comment:
codebtors=[], files=[]
never use lists or dictionaries as parameters to a function/method. 
https://docs.python-guide.org/writing/gotchas/
use something like files=None
if files is None:
  files=[]
instead.

Reply:
and there is to many parameters maybe just use **kwargs instead...